inventory/group_vars/ath/vars.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Do we need this file?

Reply:
the same reason
NOTE: remember the root cause of this change: support for multi-environ
IF THIS NOT A REQUIREMENT then all this will be needless